Source/Network/Limits.swift

```diff
@@ -8,14 +8,14 @@
 
 import Foundation
 
-struct Limits: Encodable {
+public struct Limits: Encodable {
     let duration: Duration
     
-    init(duration: Duration) {
+    public init(duration: Duration) {
```

Comment:
Probably don't need this constructor if we have the one below.

Reply:
I saw the other more as a convenience one and devs could use this one along with the `unlimited()` static func for Duration:
```
let limits = Limits(duration: .unlimited())
```

Reply:
Can we have the Duration object conform to `ExpressibleByIntLiteral`? I believe this would allow us to pass an `Int` into an call requiring a `Duration` object and it will automatically be coalesced.